test/testutil.c

```diff
@@ -73,6 +73,8 @@ int run_tests(const char *test_prog_name)
             }
         } else {
             for (j = 0; j < all_tests[i].num; j++) {
+                if (j == 625)
+                    printf("Running test 625\n");
```

Comment:
Yes...but you're reviewing the commits from #1804 that I said to ignore for this PR!